draft-ietf-quic-recovery.md

```diff
@@ -509,6 +509,58 @@ frames containing them could be lost. In this case, the loss recovery algorithm
 may cause spurious retransmits, but the sender will continue making forward
 progress.
 
+## Tracking Sent Packets {#tracking-sent-packets}
+
+QUIC stores information about every packet sent. It's expected implementations
+will index this per-packet information by packet number and store the
+per-packet fields detailed below for loss recovery and congestion control.
+Additionally, implementations MUST ensure that any retransmittable frames
+being transmitted are tracked in case of loss.
+
+If a packet containing retransmittable frames is lost, the QUIC transport
+needs to recover from that loss, such as by retransmitting the data,
+sending an updated frame, or abandoning the frame. In the case of
+STREAM data, the data should be retransmitted unless the send side of the
+stream is closed.
```

Comment:
I don't know why you would call out STREAM specially here.  Why not just reference [Section 13.2 of transport](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.13.2).

Reply:
Great idea, is this the right way to reference a section of another doc?